Sources/NostrSDK/Events/NonParameterizedReplaceableEvent.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  NonParameterizedReplaceableEvent.swift
+//
+//
+//  Created by Terry Yiu on 12/23/23.
+//
+
+import Foundation
+
+public protocol NonParameterizedReplaceableEvent: ReplaceableEvent {}
+public extension NonParameterizedReplaceableEvent {
+    func replaceableEventCoordinates(relayURL: URL? = nil) -> EventCoordinates? {
+        guard kind.isNonParameterizedReplaceable, let publicKey = PublicKey(hex: pubkey) else {
+            return nil
+        }
+
+        do {
+            return try EventCoordinates(kind: kind, pubkey: publicKey, relayURL: relayURL)
+        } catch {
+            return nil
+        }
```

Comment:
This is identical to:
```suggestion
        return try? EventCoordinates(kind: kind, pubkey: publicKey, relayURL: relayURL)
```

Essentially you're saying you don't care about the thrown error.